Assets/Scripts/Enemy/Boss/BossController.cs

```diff
@@ -192,4 +201,14 @@ private void Die()
     {
         Destroy(this.gameObject);
     }
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+        if (collision.gameObject.layer == 8)
```

Comment:
```suggestion
        if (collision.gameObject.layer == 8) // Player Projectile
```
...so it's a little easier to read at a glance what layer we're checking collision with.